static/app/views/starfish/components/releaseSelector.tsx

```diff
@@ -149,6 +149,7 @@ export function ReleaseComparisonSelector() {
 const StyledCompactSelect = styled(CompactSelect)`
   @media (min-width: ${p => p.theme.breakpoints.medium}) {
     max-width: 275px;
+    min-width: 135px;
```

Comment:
It's not as straightforward as this since we wrap our release selectors in the `<PageFilterBar>` component, and that applies a min width to the last [child](https://github.com/getsentry/sentry/blob/master/static/app/components/organizations/pageFilterBar.tsx#L62) that need to be overridden.

Right now, the min-width only gets applied to the first selector.

![Screenshot 2023-11-21 at 12 22 27 PM](https://github.com/getsentry/sentry/assets/63818634/7fb0ab51-fa11-4503-9a86-2d38f8215d20)

You could do something like this:

```
const StyledPageFilterBar = styled(PageFilterBar)`
  & > * {
    /* Prevent date filter from shrinking below 6.5rem */
    &:last-child {
      min-width: 135px;
    }
  }
`;
```

And wrap the release selectors in `<StyledPageFilterBar>` to ensure it gets applied to the last child too

Reply:
Oh yes, my bad.
Yes adding a wrapper will work fine but,

How about using **_!important_**  with min-width?